src/System.IO.FileSystem.AccessControl/src/System.IO.FileSystem.AccessControl.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <AssemblyName>System.IO.FileSystem.AccessControl</AssemblyName>
+    <ProjectGuid>{D77FBA6C-1AA6-45A4-93E2-97A370672C53}</ProjectGuid>
+  </PropertyGroup>
```

Comment:
Can you combine these property groups into just the one and put the conditions on the properties themselves.